### PR TITLE
Update instructions for unityci images

### PR DESCRIPTION
**Pull request by vtimejc**
As dicussed on Discord#docker

#### Changes

- Update GitLab documentation for license creation to match images on [unityci](https://hub.docker.com/r/unityci/editor)

#### Checklist

- [x] Read the contribution [guide](../CONTRIBUTING.md) and accept the [code](../CODE_OF_CONDUCT.md) of conduct
- [x] Readme (not needed)
- [x] Tests (not needed)

### CODE REVIEW

File: docs/gitlab/02-activation.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ All you need is [docker](https://www.docker.com/) installed on your machine.
    -e "TEST_PLATFORM=linux" \
    -e "WORKDIR=/root/project" \
    -v "$(pwd):/root/project" \
-   gableroux/unity3d:$UNITY_VERSION \
+   unityci/editor:$UNITY_VERSION \
```
Comment (GabLeRoux) [inline docs/gitlab/02-activation.md:33]: We should use something that looks like this:

```bash
   UNITY_VERSION=2020.1.11f1
   IMAGE=unityci/editor # https://hub.docker.com/r/unityci/editor
   IMAGE_VERSION=0.12 # https://github.com/game-ci/docker/releases
   DOCKER_IMAGE=$IMAGE:$UNITY_VERSION-base-$IMAGE_VERSION
```

File: docs/gitlab/02-activation.md [hunk 2]
```diff
@@ -46,9 +46,10 @@ All you need is [docker](https://www.docker.com/) installed on your machine.
 
    ```bash
    xvfb-run --auto-servernum --server-args='-screen 0 640x480x24' \
-   /opt/Unity/Editor/Unity \
+   /opt/unity/Editor/Unity \
```
Comment (GabLeRoux) [inline docs/gitlab/02-activation.md:49]: We should be using alias here

**Review by GabLeRoux — CHANGES_REQUESTED** [review]
see comments :)

File: docs/gitlab/02-activation.md (hunk 2 above)
Comment (webbertakken) [reply docs/gitlab/02-activation.md:49]: full path: `/usr/bin/unity-editor`
so: `unity-editor` should be sufficient
source: https://github.com/game-ci/docker/blob/47c7a39d8989e6c0b501405e7b9c63d3942f63e1/editor/Dockerfile#L174

**Comment by GabLeRoux** [comment]
Closing in favor of https://github.com/game-ci/documentation/pull/155 (applied review comments) 👍 🚀

**Pull request closed without merging**